src/pages/mesh/advanced/index.md

```diff
@@ -614,3 +614,115 @@ aio api-mesh:source:install "AEM Assets API"@0.0.1
 ```bash
 Successfully updated the mesh with the id: MESH_ID
 ```
+
+## aio api-mesh:log-list
+
+The `log-list` command lists the last 15 events for your mesh by rayID.
+
+### Usage
+
+```bash
+aio api-mesh:log-list
+```
+
+### Flags
+
+The following arguments are optional:
+
+`--filename` allows you to download a CSV list of rayIDs with the specified filename.
+
+`-i` or `--ignoreCache` ignores the cached organization, project, and workspace, which allows you to create a mesh in a different workspace. You can also manually [modify the cache](../basic/work-with-mesh.md#projects-and-workspaces).
```

Comment:
What's the purpose of this flag? Creating a mesh in a different workspace seems unrelated to any logging functionality. @dthampy

Reply:
@naomirobichaud  This `--i `flag isn't specifically related to logging. As you might know, the ignoreCache flag is used to allow users to switch to a different organization/project/workspace outside of what's in their local config. When used with logging commands, it lets users choose a different organization, project, or workspace from what’s defined in their local config.

Reply:
Right, I figured this was a way to switch context. Is this flag only relevant for this logging command and not the other? 

Regardless, we should remove sentence about creating a mesh in a different workspace - it's not relevant for logging and misleading. Should say that the flag allows you to choose a different mesh.

Reply:
> Is this flag only relevant for this logging command and not the other?

The ignoreCache flag can be used with all the logging commands